### DISCUSSION TITLE
Security Onion encountered "The quota has been exceeded." Error

### DISCUSSION THREAD

**Discussion by Ron89** [Security-Onion-Solutions/securityonion#7472] [Unsupported Versions]
When using Security Onion Hunt to investigate, I encountered the banner "The quota has been exceeded.". May I know what this quota is? Is additional searching locked behind some paid service? Or is there some yaml file I need to edit to change the quota? I couldn't find any related discussion, so posted the question here.

**Comment by dougburks** [comment]
Are you able to provide a screenshot?

What version of Security Onion are you using?

What browser are you using?

Have you checked the logs in /opt/so/log/ for additional clues?

**Comment by weslambert** [comment]
It might be possible that the browser's own local storage limit has been met. If using Chrome, you can try using the following Console script in Dev Tools to get an idea of what's being stored:

```
var _lsTotal=0,_xLen,_x;for(_x in localStorage){ if(!localStorage.hasOwnProperty(_x)){continue;} _xLen= ((localStorage[_x].length + _x.length)* 2);_lsTotal+=_xLen; console.log(_x.substr(0,50)+" = "+ (_xLen/1024).toFixed(2)+" KB")};console.log("Total = " + (_lsTotal / 1024).toFixed(2) + " KB");
```
